superset-frontend/cypress-base/cypress/integration/database/modal.test.ts

```diff
@@ -17,75 +17,54 @@
  * under the License.
  */
 import { DATABASE_LIST } from './helper';
-
+// TODO: Add new tests with the modal
```

Comment:
Can we assign this TODO to someone?

When I leave a TODO I write:

```
TODO (betodealmeida): do XXX
```

So I can easily `grep` for the things I'm responsible for. I've seen other people doing the same.

Reply:
I like that pattern! Fixed in [`this commit`](https://github.com/apache/superset/pull/15463/commits/83e222232aa41a7b4096b815ecff51cf2e0b8ad2).